andela/mmdp_site
# MMDP-344 Implements Fix styles

**What does this PR do?**

This PR fixes the look of the CMS

**Description of Task to be completed?**

- Fix fonts

- Fix layout

- Fix report upload button style

**How should this be manually tested?**

Check out `bg-fix-styles-MMDP-344` 
Install dependencies `yarn install`
Run server `yarn run dev`
Login to the CMS and navigate the different pages on the side bar

**Any background context you want to provide?**
Some of the current CSS styles work on browsers zoomed out to 90% 

**What are the relevant Jira issues?**

[MMDP-344](https://viisaus.atlassian.net/browse/MMDP-344)

**Screenshots (if appropriate)**

<img width="1440" alt="screen shot 2019-02-08 at 16 34 24" src="https://user-images.githubusercontent.com/31965597/52481616-0b9c0400-2bc0-11e9-8b51-e1ac9840dedf.png">

<img width="1440" alt="screen shot 2019-02-08 at 16 34 50" src="https://user-images.githubusercontent.com/31965597/52481634-15be0280-2bc0-11e9-9993-e07f1cc57551.png">

<img width="1440" alt="screen shot 2019-02-08 at 16 39 36" src="https://user-images.githubusercontent.com/31965597/52481661-253d4b80-2bc0-11e9-827b-bb996613ac9f.png">


You can have screenshots where applicable

**Questions:**
